core/src/main/java/org/neo4j/ogm/context/LabelPrimaryId.java

```diff
@@ -33,11 +35,11 @@
     /**
      * Create LabelPrimaryId
      *
-     * @param classInfo class info containign the primary id
+     * @param classInfo class info containing the primary id
      * @param id        the value of the id
      */
     public LabelPrimaryId(ClassInfo classInfo, Object id) {
-        this.label = classInfo.primaryIndexField().containingClassInfo().neo4jName();
```

Comment:
Since this change is a key aspect of this fix, I'd suggest to communicate it properly during the next release update in a way it gives enough information for those relaying on the buggy behavior.